### ISSUE TITLE
Acromioclavicular (AC) joint arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11008]
**Preferred term label:**
Acromioclavicular (AC) joint arthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Zanca view x-ray of the AC joint showing radiographic evidence of osteoarthritis such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the AC joint

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acromioclavicular (AC) joint osteoarthritis (OA)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001339
- label: Acromioclavicular joint arthritis
- parents: [Abnormal clavicle morphology (HP:0000889)](https://hpo.jax.org/app/browse/term/HP:0000889)
- pmid: PMID:23649008
- def: Degenerative disease of the acromioclavicular (AC) joint that manifests radiographically as Zanca view X-ray of the AC joint showing radiographic evidence of osteoarthritis such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the AC joint.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11008
- synonyms: Osteoarthritis of the acromioclavicular joint(NONE)
